students.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# subclass of the class person
+class Students < Person
+  def initialize(id, age, classroom, name: 'Unknown', parent_permission: true)
+    super(id, age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly see the code snippet below for how to better structure your initialize method :+1: 
- code snippet
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```